backend/src/test/java/net/es/oscars/cuke/NsoStateManagerSteps.java

```diff
@@ -54,29 +54,18 @@ public void the_state_manager_loads_VPLS_and_LSP_states() throws NsoStateManager
 
     @Given("The invalid VPLS instances don't exist")
     public void theInvalidVplsInstancesDontExists() {
-        // Clean out known invalid VPLS entries
-        Dictionary<Integer, NsoStateWrapper<NsoVPLS>> localVplsState = stateManager.getNsoVplsStateSyncer().getLocalState();
-        Dictionary<Integer, NsoStateWrapper<NsoVPLS>> remoteVplsState = stateManager.getNsoVplsStateSyncer().getRemoteState();
-
         List<String> knownInvalidVpls = new ArrayList<>() {};
 
-        knownInvalidVpls.add("BBB2");
         knownInvalidVpls.add("DOE-IN");
         knownInvalidVpls.add("DOE-IN-FNE");
         knownInvalidVpls.add("SPaRC-NMS");
         knownInvalidVpls.add("SPaRC");
-        knownInvalidVpls.add("OSCARS-XYFK");
 
+        // make sure to skip unmanaged / invalid VPLS
```

Comment:
Should this type of validation be part of the actual code base, instead of just the test?

Reply:
it is in fact part of the codebase. as of now, VPLSs with vc-id outside the defined managed range (as per `nso.vc-id-range` property) are not added to the state. similar to what we were doing with LSPs that didn't follow the OSCARS name formatting.